cinema/urls.py

```diff
@@ -1 +1,21 @@
-# write urls here
+from rest_framework import routers
+
+from cinema.views import (
+    ActorViewSet,
+    GenreViewSet,
+    CinemaHallViewSet,
+    MovieViewSet,
+    MovieSessionViewSet,
+)
+
+app_name = "cinema"
+
+router = routers.DefaultRouter()
+router.register("actors", ActorViewSet)
+router.register("genres", GenreViewSet)
+router.register("cinema_halls", CinemaHallViewSet)
+router.register("movies", MovieViewSet)
+router.register("movie_sessions", MovieSessionViewSet)
+
+
+urlpatterns = [] + router.urls
```

Comment:
There is no need to include `[] +` part here, you could assign `router.urls` directly to `urlpatterns` instead in the future